### ISSUE TITLE
/learn feedback

### ISSUE THREAD

**Issue by joepio** [joepio/pauseai#424]
From Alvaro Shifu.

Readability of white text on orange BG is insufficient. We should consider having an automated WCAG check.

<img width="1184" height="916" alt="Image" src="https://github.com/user-attachments/assets/13a73d83-3246-4f46-88ef-bcc9c25845a3" />

**Comment by conchafetida** [comment]
Yes, mainly, and also making a big focus on font line widths to make very sure the site is fully readable. We can automate it.
